### ISSUE TITLE
Packer 0.9.0 - vmware-esxi builder & features

### ISSUE THREAD

**Issue by shashanksinha1989** [mitchellh/packer#3513]
I am looking forward to use these featues 

With packer version 0.9.0 "vmware-esxi" featues were released. 

1. How can I utilize these features, is there is any documentation for the same. vmware-esxi ?

2. Do we have vmware-esxi as a new builder as it is mentioned in improvement section for 0.9.0 packer version ?

**Comment by rickard-von-essen** [comment]
Please only use _GitHub Issues_ for reporting bugs and feature requests. Do not ask for general help here. Use _IRC `#packer-tool` on Freenode_ or the [mailing list](https://groups.google.com/group/packer-tool) for that.

See https://www.packer.io/community/

_If you are describing a bug or a feature request please reopen and try to add some more details to make it clearer._

**Comment by rickard-von-essen** [comment]
It's documented in:
https://www.packer.io/docs/builders/vmware-iso.html
https://www.packer.io/docs/builders/vmware-vmx.html

See `remote_*` and **Building On A Remote VSphere Hypervisor**. If you need more guidance please try IRC or the mailing list. 

And please use the latest release _v0.10.1

**Comment by shashanksinha1989** [comment]
Thanks, I'll ask in IRC as specifically I am looking to feature to export images from vcloud

**Comment by shashanksinha1989** [comment]
@rickard-von-essen I had tried both google groups and IRC for my query but so far no luck. The document links mentioned above doesn't explain how I can import images from vCloud or vSphere. This feature was released with packer version 0.9.0

![image](https://cloud.githubusercontent.com/assets/13942201/15275936/a1f04f48-1af6-11e6-9ea1-716a2e0d8955.png)

If not is there a possibility to build VM from cloned VM on remote VSphere Hypervisor? As the feature released within this version will solve my problem.
